contracts/NodeDelegator.sol

```diff
@@ -17,6 +19,12 @@ import { ReentrancyGuardUpgradeable } from "@openzeppelin/contracts-upgradeable/
 import { IEigenPodManager } from "./interfaces/IEigenPodManager.sol";
 import { IEigenPod, BeaconChainProofs } from "./interfaces/IEigenPod.sol";
 
+struct ValidatorStakeData {
```

Comment:
Why is this outside of the contract?

Reply:
It makes it much easier when testing using Forge. The `ValidatorStakeData` struct can be imported from the `NodeDelegator.sol` file and used. The alternative is `NodeDelegator` is imported and `NodeDelegator.ValidatorStakeData` is used in the tests.